cosmpy/crypto/hashfuncs.py

```diff
@@ -21,15 +21,8 @@
 
 import hashlib
 
-from _hashlib import HASH  # type: ignore  # pylint: disable=no-name-in-module
-
-# Detect if ripemd160 can actually be used in the system. Querying `hashlib.algorithms_available`
-# does not mean much and will fail on 22.04 LTS
-try:
-    hashlib.new("ripemd160")
-    _ripemd160_present = True
-except ValueError:
-    _ripemd160_present = False
+# pycryptodome, a dependency of bip-utils
+from Crypto.Hash import RIPEMD160  # type: ignore # nosec
```

Comment:
Need to test this on an M1/M2 Mac. I think there was a problem on an M1 Mac which lead to the existing solution.